tests/cross/src/test/scala/tests/pc/InlayHintsSuite.scala

```diff
@@ -853,4 +853,24 @@ class InlayHintsSuite extends BaseInlayHintsSuite {
     ),
     hintsInPatternMatch = true
   )
+
+  check(
+    "apply-param".tag(IgnoreScala2),
+    """|object Main:
+       |  case class A()
+       |  case class B[T]()
+       |  given A = A()
+       |  implicit def bar(using a: A): B[A] = B[A]()
+       |  def foo(using b: B[A]): String = "aaa"
+       |  val g: String = foo
+       |""".stripMargin,
+    """|object Main:
+       |  case class A()
+       |  case class B[T]()
+       |  given A = A()
+       |  implicit def bar(using a: A): B[A] = B[A]()
+       |  def foo(using b: B[A]): String = "aaa"
+       |  val g: String = foo/*(using bar<<(5:15)>>)*/
```

Comment:
How would it look before actually? Should we have a feature request with this information being available somehow?

Reply:
No, basically, it will just show `bar` again, since that is the implicit conversion for `given_A`.
```
val g: String = foo/*)*//*bar<<(5:15)>>(*//*(using bar<<(5:15)>>)*/
```